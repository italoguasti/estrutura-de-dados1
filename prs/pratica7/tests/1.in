10
jose 1 5 1990
guilherme 9 3 1998
valentina 1 3 1945
jesus 25 1 0000
pedro 19 10 1930
maria 17 5 1972
giovana 8 3 2015
siva 3 11 1968
daiane 24 12 1993
paulo 6 4 1998
