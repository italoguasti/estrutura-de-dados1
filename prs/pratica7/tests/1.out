Analise.....: Comp | Movi
- Bubble....: 0045 | 0017
- Selection.: 0045 | 0007
- Insertion.: 0026 | 0026
